src/data/monsters.txt

```diff
@@ -2333,35 +2333,17 @@ Boltsmann security robot	2402	jetsecurity1.gif	Scale: 9 Cap: 1000 Init: -10000 P
-gooified flower	2223	goo_flower.gif	Atk: 180 Def: 180 HP: 300 Init: -10000 P: plant Article: a	gooified vegetable matter (100)	gooified vegetable matter (100)	gooified vegetable matter (100)
-gooified tree	2224	goo_tree.gif	Atk: 150 Def: 50 HP: 1000 Init: -10000 P: plant Article: a	gooified vegetable matter (10)	gooified vegetable matter (10)	gooified vegetable matter (10)	gooified vegetable matter (10)	gooified vegetable matter (10)
-gooified rock slab	2225	goo_slab1.gif,goo_slab2.gif,goo_slab3.gif	Atk: 250 Def: 250 HP: 500 Init: -10000 P: elemental Article: a	1-6 gooified mineral matter (m100)
-massive goo construct	2226		Atk: 100 Def: 100 HP: 100 Init: -10000 P: construct EA: hot EA: sleaze Article: a	4-20 gooified animal matter (m100)	4-20 gooified vegetable matter (m100)	4-20 gooified mineral matter (m100)
-
-# Crimbo 2022
-Brake-Operating Trainbot	2254	trainbot_brake.gif	NOWISH Scale: 0 Cap: ? Init: -10000 P: construct Article: a
-Track-Switching Trainbot	2255	trainbot_switch.gif	NOWISH Scale: 0 Cap: ? Init: -10000 P: construct Article: a
-Ping-Pong-Playing Trainbot	2256	trainbot_ping.gif	NOWISH Scale: 0 Cap: ? Init: -10000 P: construct Article: a
-Drink-Delivery Trainbot	2257	trainbot_drinks.gif	NOWISH Scale: 0 Cap: ? Init: -10000 P: construct EA: hot EA: cold Article: a
-Luggage-Handling Trainbot	2258	trainbot_bags.gif	NOWISH Scale: 0 Cap: ? Init: -10000 P: construct EA: hot Article: a
-Ticket-Checking Trainbot	2259	trainbot_check.gif	NOWISH Scale: 0 Cap: ? Init: -10000 P: construct EA: hot Article: a
-Table-Waiting Trainbot	2260	trainbot_wait.gif	NOWISH Scale: 0 Cap: ? Init: -10000 P: construct Article: a
-Wine-Pairing Trainbot	2261	trainbot_wine.gif	NOWISH Scale: 0 Cap: ? Init: -10000 P: construct EA: sleaze Article: a
-Table-Bussing Trainbot	2262	trainbot_bus.gif	NOWISH Scale: 0 Cap: ? Init: -10000 P: construct Article: a
-Coal-Shoveling Trainbot	2263	trainbot_coal.gif	NOWISH Scale: 0 Cap: ? Init: -10000 P: construct EA: hot Article: a
-Slag-Processing Trainbot	2264	trainbot_slag.gif	NOWISH Scale: 0 Cap: ? Init: -10000 P: construct EA: hot Article: a
-Steam-Routing Trainbot	2265	trainbot_steam.gif	NOWISH Scale: 0 Cap: ? Init: -10000 P: construct EA: hot Article: a
-The Superconductor	2266	superconductor.gif	BOSS NOCOPY NOMANUEL Scale: 30 Cap: ? HP: 300 Init: -10000 P: construct EA: hot	The Superconductor's CPU (100)
+# Crimbo 2023 (December 2023)
+Crimbuccaneer carpenter	2432	cbcarpenter.gif
+Crimbuccaneer freebooter	2430	cbboots.gif
+Crimbuccaneer military school dropout	2405	cbdropout.gif
+Crimbuccaneer new recruit	2395	cbrecruit.gif
+Crimbuccaneer privateer	2436	cbprivateer.gif
+Crimbuccaneer scrimshander	2431	cbscrimshander.gif
+Crimbuccaneer whalehunter	2428	cbwhaler.gif
+Elf Guard conscript	2392	egconscript.gif
+Elf Guard convict	2393	egconvict.gif
+Elf Guard private	2394	egprivate.gif
```

Comment:
we normally sort these by ID, instead of alphabetically

Reply:
It's really mixed, but the vast majority of the page is alphabetically sorted (see Regular monsters section)

Reply:
the regular monsters section is alphabetical, but new areas tend to be id. I like it because it tells you whether you've missed one (e.g., path bosses).

It's okay to keep this how you like it.

Reply:
I like new areas to be sorted by ID since, as Chris said, it's a clue that you might be missing some.